Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [astr-i18n/ASTR](https://hosted.weblate.org/projects/astr-i18n/astr/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/astr-i18n/astr/horizontal-auto.svg)

30000lightyears committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 97.1% (67 of 69 strings)

Translation: astr-i18n/ASTR
Translate-URL: https://hosted.weblate.org/projects/astr-i18n/astr/zh_Hans/

c-stella committed:
Translated using Weblate (Japanese)

Currently translated at 79.7% (55 of 69 strings)

Translation: astr-i18n/ASTR
Translate-URL: https://hosted.weblate.org/projects/astr-i18n/astr/ja/

050644zf commented:
@c-stella we have more string that updated just now

c-stella commented:
@050644zf 
~~Gotcha, will take a look at it when I got a spare time.~~
... Apparently there wasn't much new words, so I've sent suggestions.

c-stella committed:
Translated using Weblate (Japanese)

Currently translated at 85.1% (63 of 74 strings)

Translation: astr-i18n/ASTR
Translate-URL: https://hosted.weblate.org/projects/astr-i18n/astr/ja/

050644zf commented:
@c-stella look good to me, thank you for your contribution 🤝
